pkg/client/unversioned/clientcmd/api/types.go

```diff
@@ -88,6 +88,8 @@ type AuthInfo struct {
 	ClientKeyData []byte `json:"client-key-data,omitempty"`
 	// Token is the bearer token for authentication to the kubernetes cluster.
 	Token string `json:"token,omitempty"`
+	// TokenFile is a pointer to a file that contains a bearer token (as described above).  If both Token and TokenFile are present, TokenFile takes precedence.
```

Comment:
I don't super care about which precendence is used.

@smarterclayton seems to indicate that `tokenFile` should take precedence in his comment here:

https://github.com/kubernetes/kubernetes/issues/28260#issuecomment-230060072